index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    
+    <section class="first-page">
+        <header>
+            <nav id="head">
+                <a href="#">Safar Ali</a>
+                <a href="#"><img src="./images/Union.png" alt="burger-icon"></a>
+            </nav>
+        </header>
+        <section class="headline">
+            <div id="name">
+                <h1>I'm Safar Ali <br> A thriving <br> Software Developer </h1>
```

Comment:
## OPTIONAL

- [ ] Kindly remove all br tags and use margin for styling of text, this also helps you understand[ CSS box model
](https://www.w3schools.com/css/css_boxmodel.asp)  and adjust h1 tag to look like figma design 👍🏿